tool/filter_css_tables.py

```diff
@@ -0,0 +1,18 @@
+################################################
+## Add class to tables
+## Author: Rome Reginelli
```

Comment:
Hey, take credit for your own work here! :laughing:

Reply:
haha done